tests/test_claim_pipeline_v2.py

```
@@ -0,0 +1,221 @@
+    assert inner.draft_inputs == ["RAW CASE"]
+    assert draft.title == "Исковое заявление"
+    assert pretrial.status == VerificationStatus.VERIFIED
+    assert inner.other_calls == 1
+
+
+def test_active_augments_only_research_and_keeps_raw_draft_context(monkeypatch) -> None:
+    monkeypatch.setenv("KORGAN_CLAIM_PIPELINE_V2_MODE", "active")
+    inner = _FakeInner()
+    adapter = ClaimPipelineV2Adapter(inner)
+
+    async def build_packet(case_context: str, language: str) -> ClaimPipelinePacket:
```

Ruff (0.16.2)

[warning] Unused function argument: `language`

(ARG001)